build.gradle

```diff
@@ -0,0 +1,55 @@
+apply plugin: 'java'
+apply plugin: 'idea'
+apply plugin: 'maven-publish'
+
+group 'org.logstash.beats'
+version "${new File("VERSION").text.trim()}"
+
+description = "Beats/Lumberjack Netty implementation"
+
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testCompile 'junit:junit:4.12'
+    testCompile 'org.hamcrest:hamcrest-library:1.3'
+
+    compile 'org.bouncycastle:bcprov-jdk15on:1.54'
+    compile 'org.bouncycastle:bcpkix-jdk15on:1.54'
+    compile 'io.netty:netty-all:4.1.1.Final'
+    compile 'io.netty:netty-tcnative-boringssl-static:1.1.33.Fork17'
```

Comment:
cool, then we just need to release a new plugin version, if there is another sslpocalypse (hopefully much less often with boringssl)